File: 1º Semestre/Programação de Computadores/Lista de Exercicios 1/12.cpp
#include <iostream>
using namespace std;

//12. Reescrever o programa anterior apresentando o quadrado e o cubo do número informado

int main(){
	float numero;
	
	printf("Digite um numero real: ");
	scanf("%f", &numero);
	
	printf("\nNumero -> %0.2f", numero);
	printf("\nDobro deste numero -> %0.2f", numero * 2);
	printf("\nQuadrado deste numero -> %0.2f", numero * numero);
	printf("\nCubo deste numero -> %0.2f", numero * numero * numero);
}

File: 1º Semestre/Programação de Computadores/Lista de Exercicios 1/Python/12.py
"""
12.	Reescrever o programa anterior apresentando o quadrado e o cubo do número informado.
"""

floatInput = float(input("Digite um numero real: "))
print("""
Numero -> {:.0f}
Dobro deste numero -> {:.0f}
Quadrado deste numero -> {:.0f}
Cubo deste numero -> {:.0f}
""".format(floatInput, floatInput * 2, floatInput ** 2, floatInput ** 3))